src/service/viewport-impl.js

```diff
@@ -652,8 +671,11 @@ export class ViewportBindingDef {
   /**
    * Updates binding with the new padding.
    * @param {number} unusedPaddingTop
+   * @param {boolean|undefined} unusedOptUpdateScrollPos
```

Comment:
I don't think we can just use `lastPaddingTop`. But one option is that we can calculate and use delta. However, given the current uncertainty over the exact rules, I'd rather focus on that and clean it up once we have everything working correctly. E.g. on iOS the formula may end up looking something like `newPadding * 2 - oldPadding / 2`.